Header header
float64 angle    # 雷达角度（0-360度）
